src/Tribe/Validate.php

```diff
@@ -499,6 +499,38 @@ public function email(  ) {
 			}
 		}
 
+		/**
+		 * Validates and sanitizes a list of email addresses.
+		 *
+		 * @since TBD
+		 */
+		public function email_list() {
+			$candidate = trim( $this->value );
+			$emails = preg_split( '[,|;]', $candidate );
```

Comment:
I think we should break this method down a bit further.

Reply:
Made a few tweaks after doing a little more testing with it. Added some more comments, too. What did you have in mind as far as breaking it down?

Reply:
Pinging you again on this @bordoni

Reply:
One more try @bordoni ?